moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -153,13 +184,14 @@ export const useColumns = ({ hiddenColumns, classes }) => {
         field: "project_name",
         hidden: hiddenColumns["project_name"],
         render: (entry) => (
-          <RouterLink
+          <Link
+            component={RouterLink}
             to={`/moped/projects/${entry.project_id}`}
             state={{ queryString }}
-            className={classes.colorPrimary}
```

Comment:
I removed this className so we would have one less param to consume in this hook. One result is that we go back to [the default MUI behavior of having underlined links](https://github.blog/changelog/2023-10-18-new-default-underlined-links-for-improved-accessibility/) which might not be the worst thing for accessibility. [Github recently did this too.](https://github.blog/changelog/2023-10-18-new-default-underlined-links-for-improved-accessibility/)